src/api/entities/Checkpoint.ts

```diff
@@ -101,6 +101,7 @@ export class Checkpoint extends Entity<UniqueIdentifiers, HumanReadable> {
   /**
    * Retrieve all Asset Holder balances at this Checkpoint
    *
+   * @note current Asset holders who didn't hold any tokens when the Checkpoint was created will be listed with a balance of 0
```

Comment:
Maybe sprinkle "due to chain limitations" at the beginning to make it clear that this isn't an arbitrary senseless decision